### PR TITLE
Update inferCNV to v0.8.2

### PR DESCRIPTION
**Pull request by eweitz**
This updates Single Cell Portal's inferCNV Docker image and WDL to [inferCNV v0.8.2](https://github.com/broadinstitute/inferCNV/releases/tag/InferCNV-v0.8.2).  

You can pull it like so:
`docker pull singlecellportal/infercnv:0.8.2-rc1`

The WDL previously only existed in the [FireCloud Methods Repository](https://portal.firecloud.org/#methods).  

This also renames the Dockerfile so that it can be easily detected by syntax highlighters (I use VS Code), and removes the tarball for the old version.

As part of this work I experimented with different base images.  Some statistics:

* ubuntu:xenial: 1.48 GB image, 88 minutes to build
* r-ver: 1.52 GB image, 50 minutes to build

So we get a 40 MB smaller image for a 38 minutes-longer initial build.

At first glance that seems to over-favor the Docker image user, but for the Docker image developer, waiting ~1 hour doesn't feel drastically shorter than waiting ~1.5 hours.  And that's only for the very first, completely uncached build.  So I'm inclined to save the user a few seconds of workflow run time for every submission.

This workflow is not finished; subsequent PRs will be sent for that.  This PR only concerns updating inferCNV to v0.8.2.

### CODE REVIEW

File: WDL/infercnv/infercnv.wdl [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        memory: "8 GB"
+        bootDiskSizeGb: 12
+        disks: "local-disk ${diskSpace} HDD"
+        cpu: 8
+        preemptible: 2
+    }
+}
+
+task run_matrix_to_ideogram_annots {
+	File matrix_path
```
Comment (jlchang) [inline WDL/infercnv/infercnv.wdl]: extra indent here?

Comment (eweitz) [reply WDL/infercnv/infercnv.wdl]: Fixed!

**Comment by eweitz** [comment]
> The reasoning Tim shared about the "for_/Dockerfile" nomenclature was that Docker files don't "work" when called anything other than plain Dockerfile - so that's my only objection to calling the file infercnv.dockerfile

The `-f` argument enables building Docker images with files not named `Dockerfile`.  For example, here's the command I ran to build this docker file:

`docker build --tag infercnv:0-8-2-rc1 -f infercnv.dockerfile .`

Using the typical Dockerfile name would certainly make sense if we ever re-organize our directory layout to be more workflow-centric rather than file-type centric, e.g. infercnv/infercnv.wdl, infercnv/Dockerfile, etc. vs. WDL/infercnv.wdl, docker_files/infercnv.dockerfile, etc.

File: docker_files/infercnv.dockerfile [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Docker file for inferCNV
+FROM ubuntu:xenial
+MAINTAINER ttickle@broadinstitute.org
```
[the remark this answers is not in the document]
Comment (eweitz) [reply docker_files/infercnv.dockerfile]: Done.

**Comment by eweitz** [comment]
Thanks for the review and context!

**Pull request merged**